e2e/specs/popup.test.js

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+'use strict';
+
+const fs = require('fs');
+
+const robot = require('robotjs');
+
+const env = require('../modules/environment');
+const {asyncSleep} = require('../modules/utils');
+
+describe('popup', function desc() {
+    this.timeout(40000);
+
+    const config = env.demoMattermostConfig;
+
+    beforeEach(async () => {
+        env.cleanDataDir();
+        env.createTestUserDataDir();
+        env.cleanTestConfig();
+        fs.writeFileSync(env.configFilePath, JSON.stringify(config));
+        await asyncSleep(1000);
```

Comment:
Out of curiosity what is this sleep for?

Reply:
Just to make sure that the file has been written before we attempt to start the app. I know it's done synchronously but I ran into situations in the past where the config wasn't there, so it's just to make sure it is.